test/NeinLinq.Tests/SelectorTranslator/ApplyTest.cs

```diff
@@ -44,5 +44,15 @@ public void ShouldMergeSelectors()
                 v => { Assert.Equal(2, v.Id); Assert.Equal("Narf", v.Name); },
                 v => { Assert.Equal(3, v.Id); Assert.Equal("Qwer", v.Name); });
         }
+
+        [Fact]
+        public void ShouldNotThrowOnEmptyNewExpression()
+        {
+            Expression<Func<Dummy, DummyView>> s = d => new DummyView();
+            Expression<Func<Dummy, DummyView>> t = d => new DummyView();
```

Comment:
We need two more tests here.

d => new DummyView { ... } and d => new DummyView() vs. the other way round.